Package.swift

```diff
@@ -6,7 +6,7 @@ import PackageDescription
 let package = Package(
   name: "KarrotFlex",
   platforms: [
-    .iOS(.v9),
+    .iOS(.v11),
```

Comment:
https://github.com/layoutBox/FlexLayout/blob/ae3cd19be1eefbaef939149d79c5fc228c66e131/FlexLayout.podspec#L10 FlexLayout supports 9.0, how about sync with FlexLayout iOS version?

Reply:
`swift-snapshot-testing` supports 11.0  🥲
First of all, the swift package was made to be supported from 11.

Reply:
If KarrotFlex has more users in the future, we should be careful about this situation. 
Okey I understand. 👍